src/main/java/clara/benchmark/interop/AccumRepeatedlyNotSelected.java

```diff
@@ -0,0 +1,50 @@
+package clara.benchmark.interop;
+
+import clara.rules.RuleLoader;
+import clara.rules.WorkingMemory;
+import clara.rules.QueryResult;
+import clara.benchmark.beans.Customer;
+import clara.benchmark.IBenchmark;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.lang.Iterable;
+
+public class AccumRepeatedlyNotSelected implements IBenchmark<WorkingMemory> {
+
+    //@Override
+    public String getName() {
+        return "clara.accum-repeatedly-not-selected";
+    }
+
+    //@Override
+    public WorkingMemory getSession() throws Exception {
+        return RuleLoader.loadRules("clara.benchmark.accum-repeatedly-not-selected");
+    }
+
+    //@Override
+    public Iterable<?> getFacts() throws Exception {
+        List<Customer> customers = new ArrayList<Customer>();
+        for (int i = 1; i < 100; i++) {
+            customers.add(new Customer(i, "CUSTOMER_NAME", true));
+        }
+        return customers;
+    }
+
+    //@Override
+    public void run(WorkingMemory session, Iterable<Object> facts) throws Exception {
+        WorkingMemory lastSession = session;
+
+        for (Object fact : facts) {
+            List<Object> toInsert = new java.util.ArrayList<Object>();
```

Comment:
Why are you building an ArrayList here?  Everything happening in this run() method is being timed by the benchmarking, so this is going to throw off your numbers if you are performing more overhead here than in the Drools case.  I think this is going to hurt your performance numbers.  

I don't fully understand why you have chosen to do a insert + fire every iteration.  Why not insert all facts then fire in both the Clara and Drools case?  I think that'd be a clearer indication of a realistic workflow to demonstrate accumulators not changing result values is more performant now and to contrast with how Drools is currently handling this case.